### ISSUE TITLE
[@uifabric/example-app-base] Header navigation is not accessible when zoomed in

### ISSUE THREAD

**Issue by lucas-stanford** [microsoft/fluentui#16722]
### Environment Information
Version:  2004 (OS Build: 20236.1005)
Browser: Edge chromium 
Bug id: 8708964

### Describe the issue:
In the package `@uifabric/example-app-base`, The top right navigation disappears and is not accessible in anyway when the browser width is too small or zoomed in too much.

### Please provide a reproduction of the issue in a codepen:
N/A issue with `@uifabric/example-app-base`

#### Actual behavior:
The top right navigation disappears and is not accessible when zoomed in.

#### Expected behavior:
That the navigation is accessible in some manner when zoomed in or on a small screen.

### Documentation describing expected behavior
Normal zoom
![image](https://user-images.githubusercontent.com/3011776/106345570-109f6580-6266-11eb-88e3-8b29a5dbcdd2.png)

400% zoom
![image](https://user-images.githubusercontent.com/3011776/106345577-1f861800-6266-11eb-82c8-236e15b34e71.png)

**Comment by gouttierre** [comment]
@lucas-stanford - thank you for filing this item with us. 

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue?

cc @smhigley

**Comment by tyveille** [comment]
@khmakoto @smhigley 

Are there any updates for this issue?
